sdk/tests/conformance/extensions/ext-disjoint-timer-query.html

```diff
@@ -58,6 +62,15 @@
         testPassed("No EXT_disjoint_timer_query support -- this is legal");
     } else {
         runSanityTests();
+
+        // Clear disjoint value.
+        gl.getParameter(ext.GPU_DISJOINT_EXT);
+
+        runElapsedTimeTest();
+        runTimeStampTest();
+
+        gl.finish();
+        setTimeout(checkQueryResults, 0);
```

Comment:
Also let's please retry fetching the query's availability and result for some more reasonable period like half a second (500 ms) -- 16 ms might be optimistic.